docs/platform/concepts/backup-to-another-region.rst

```diff
@@ -0,0 +1,79 @@
+About backup to another region for Aiven services
+=================================================
+
+.. important::
+
+    Backup to another region (BTAR) is available on ``:doc:Pro Platform </docs/platform/concepts/pro-platform>``.
+
+Discover the backup to another region (BTAR) feature, check why use it, and learn how it works.
+
+About BTAR
+----------
+
+The backup to another region (BTAR) feature allows existing backup files to be copied from the service's primary backup region to one or more regions different from the primary one. As a disaster recovery feature, BTAR is particularly useful when the service-hosting region is down since it allows forking the service from an additional copy of the backup residing outside the service-hosting region.
+
+BTAR is currently supported for the following services:
+
+* Aiven for PostgreSQL®
+* Aiven for MySQL®
+
+Why use BTAR
+------------
+
+BTAR provides an additional level of security for your data. It is a cost-efficient low-complexity disaster recovery feature that allows you to spread your data across different regions and recovery it in case of data loss or corruption in the primary region. During an outage or a downtime on the primary region, your data is protected and safe in a secondary backup location.
+
+How BTAR works
+--------------
+
+By enabling the backup to another region (BTAR) feature, you create an additional backup on top of the default one located in a primary region where your service is hosted. Contrary to the primary backup, the secondary backup is located outside the primary region.
+
+Secondary backups are taken from primary backups, not from the service itself. For this reason, your secondary backup is available only after a primary backup is there. There might be a data replication lag between the primary region and the secondary region.
+
+If you want to have your service restored from a secondary backup, for example, in case of an outage of the primary region, you create a fork of the service using a region where your secondary backup is located.
```

Comment:
```suggestion
Restoring from a secondary backup, for example in case of an outage of the primary region, can be done by creating a fork of the service in the region where the secondary backup is located.
```